corehq/apps/hqwebapp/static/hqwebapp/js/hqModules.js

```diff
@@ -39,14 +40,50 @@
 
 var COMMCAREHQ_MODULES = {};
 
-function hqDefine(path, moduleAccessor) {
-    if (typeof COMMCAREHQ_MODULES[path] !== 'undefined') {
-        throw new Error("The module '" + path + "' has already been defined elsewhere.");
-    }
-    if (path.match(/\.js$/)) {
-        throw new Error("Error in '" + path + "': module names should not end in .js.");
+/*
+ * Transitional version of "define" to handle both RequireJS and non-RequireJS pages.
+ * Signature deliberately matches that of "define". On non-RequireJS pages, the dependencies
+ * argument is optional, and moduleAccessor gets passed jQuery, knockout, and underscore, in that order.
```

Comment:
> moduleAccessor gets passed jQuery, knockout, and underscore, in that order.

Is that true? I don't see where it happens.

Reply:
yep, this is outdated, will fix